Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -8,33 +8,70 @@
 Installation on Google GKE
 **************************
 
+GKE Requirements
+================
+
+1. Install the Google Cloud SDK (``gcloud``)
```

Comment:
Do we really need to duplicate this or can we just refer to https://cloud.google.com/kubernetes-engine/docs/quickstart?